src/Application/Project/Project.cc

```diff
@@ -592,20 +592,20 @@ bool ProjectPrivate::initialize_session_database()
 
   // Create table for storing the database version
   sql_statements += "CREATE TABLE database_version "
-    "(version INTEGER NOT NULL PRIMARY KEY);";
+    "(version INTEGER NOT nullptr PRIMARY KEY);";
```

Comment:
These are SQL statements so they need to be reverted back to using NULL, not nullptr. Bad "replace all" bug introduced here...